framework/src/modules/nft/method.ts

```diff
@@ -347,12 +310,10 @@ export class NFTMethod extends BaseMethod {
 				NftEventResult.RESULT_NFT_DOES_NOT_EXIST,
 			);
 
-			throw new Error('NFT substore entry does not exist');
+			throw new Error('NFT does not exist');
```

Comment:
If the goal of this try/catch was simply to log the event, then how about simply re-throwing the same error, instead of creating a new one?

Same for the other instances of this pattern in the code.

Reply:
The errors are following the LIP and make sense for me - the errors inside `getNFT` are more granular depedning upon which substore entry is missing. Inside higher level functions such as `lock` it makes sense to encapsualte both into a single generic error instead of 2 different errors.